app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -4,7 +4,6 @@ import androidx.arch.core.executor.testing.InstantTaskExecutorRule
 import com.google.common.truth.Truth.assertThat
```

Comment:
We avoid using the `.kt` extensions in commit messages. In addition, instead of writing the method name, we should prefer writing a short description of what the method does.

Can you reword this commit message to something like `Add method to fetch recent patients to PatientRepository`?